requirements.txt

```diff
@@ -8,3 +8,4 @@ opencv-python==4.4.0.42
 matplotlib
 numpy
 requests
+pycocotools @ git+https://github.com/philferriere/cocoapi.git@2929bd2ef6b451054755dfd7ceb09278f935f7ad#subdirectory=PythonAPI
```

Comment:
Does this auto-install COCO API for python 3 or is more setup needed? I've setup COCO API on my local but might be useful to know for those who haven't

Reply:
I'm hoping it auto-installs with `pip3 install -r requirements.txt`. This is just what my `pip3 freeze` showed. Not sure if it carries over to new machines